test/verify/check-image

```diff
@@ -548,8 +548,62 @@ class TestImage(composerlib.ComposerCase):
         # collect code coverage result
         self.check_coverage()
 
-    @unittest.skipIf(os.environ.get("TEST_OS") == "fedora-31", "Does not support ostree image")
-    def testOSTree(self):
+    def testCancel(self):
```

Comment:
Note: The diff looks weird. Conceptually I didn't move `testCancel()`, but `testOSTree()` into a new class below, but apparently testCancel() is shorter :grin: